algoliasearch-core/src/main/java/com/algolia/search/models/rules/RuleQuery.java

```diff
@@ -16,7 +16,7 @@
   private String query;
   private String anchoring = null;
   private String context = null;
-
+  private Boolean enabled;
```

Comment:
Shouldn't we add a default value of `true`?

Reply:
According to the doc `enabled` is optional: https://www.algolia.com/doc/api-reference/api-methods/search-rules/#method-param-enabled

So a `nullable` Boolean does the trick :)

That's also what we have in C#:  

https://github.com/algolia/algoliasearch-client-csharp/blob/a50b5330e747aa3ba8f735d9296fe542d6f0d6c9/src/Algolia.Search/Models/Rules/RuleQuery.cs#L69

Reply:
Ok got it 👍 Then should we make it `private Boolean enabled = null;`?

Reply:
`private Boolean enabled;` is redundant with `private Boolean enabled = null;`
Others parameters like `private String context = null;` are like this because I C/C from the `v2`. I could update them to be consistent what do you think?

Reply:
Ok, thank you for the context. Then yes, we should probably remove the `= null` in that case. This is what confused me in the first place, and it will probably bite us one more time in the future.

Reply:
Fixed with 74d33b2. Let me know if it's good for you 👍